cli/src/helpers/InputOutput.ts

```diff
@@ -4,9 +4,13 @@ import ExitCodes from '../ExitCodes'
 import fs from 'fs'
 import path from 'path'
 import Ajv from 'ajv'
-import { JSONSchema7 } from 'json-schema'
+import $RefParser, { JSONSchema } from '@apidevtools/json-schema-ref-parser'
+import { getSchemasLocation } from 'cd-schemas'
 import chalk from 'chalk'
 
+// Default schema path for resolving refs
+const DEFAULT_SCHEMA_PATH = getSchemasLocation('entities') + path.sep
```

Comment:
Here we are assuming that the schemas we have at this location match the schemas on chain.
That may not always a valid expectation and the cli should probably not rely on them, but instead read them/reconstruct them from the chain.

Reply:
Also on this point, if cli package depends on the cd-schemas package, we would have to make sure to publish that to npm as well. Is it something we want to do?

Reply:
I'm not sure if this still applies after https://github.com/Joystream/joystream/pull/1530, but:

1. I think it would be very useful to have `cd-schemas` published either way. Perhaps it can become part of `@joystream/js` if I manage to get into that before the release. If not, it can be just a standalone library that makes interactions with `content-directory` runtime module easier.

1. > Here we are assuming that the schemas we have at this location match the schemas on chain.

Since this is used by higher-level commands like `uploadVideo` etc. that make some assumptions about the `Video` schema anyway (when populating `VideoEntity` interface with data provided by the user) I think it may be acceptable? 

The alternative would be to avoid any assumptions about classes/schemas at all in the CLI and just have a command like `createEntity Video 0` (where `0` would be `schemaIndex`), This was the idea behind https://github.com/Joystream/joystream/issues/1441 which I postponed a bit due to change of priorities, but I hope we'll have both of those solutions implmented soon and we could then compare their pros and cons (for now I think having a nice, customizable UI for uploading a video is a good thing, because CLI will temporarly be the only place where this could be done)